deployment/cfn/template_utils.py

```diff
@@ -0,0 +1,113 @@
+import ConfigParser
+import boto.ec2
+import majorkirby as mj
+
+VPC_CIDR = '10.0.0.0/16'
+ALLOW_ALL_CIDR = '0.0.0.0/0'
+
+EC2_REGIONS = [
```

Comment:
Actually, couldn't find this being used anywhere (either on develop or this branch) so I'm just going to delete it.